### PR TITLE
WIP: added cookie banner and policy with OneTrust trial

### PR DESCRIPTION
**Pull request by trabaldini**
Used Optanon by OneTrust for this, free trial for 30 days. The license costs £345 plus VAT per year. 
You can edit the styling of the banner, change the consent notice and text. Cookies are assigned automatically but can also be reassigned. Different languages are available. You can create a schedule for the audits and change the period between audits.

Seems like a good tool and easy to use.
Tried also some tools for free, but it's a lot of work more, they don't update automatically and they are mostly not covering (for sure) everything regarding the law.

### CODE REVIEW

**Comment by marcoow** [comment]
> The license costs £345 plus VAT per year.

Is that per domain or for unlimited domains? Seems a bit expensive for something so limited. Also have we confirmed we are actually required to have a cookie banner?

**Comment by trabaldini** [comment]
Used this site for the privacy policy: http://www.privacypolicyonline.com/
For almost every other site I visited you have to pay as a business

File: imprint/index.html [hunk 1]
```diff
@@ -31,12 +31,49 @@
...
+          <p> <b>Cookies and Web Beacons</b><br>https://simplabs.com uses cookies to store information about visitors' preferences, to record user-specific information on which pages the site visitor accesses or visits, and to personalize or customize our web page content based upon visitors' browser type or other information that the visitor sends via their browser. </p> 
+          <p><b>DoubleClick DART Cookie</b><br> 
+           
+          &rarr; Google, as a third party vendor, uses cookies to serve ads on https://simplabs.com.<br> 
+          &rarr; Google's use of the DART cookie enables it to serve ads to our site's visitors based upon their visit to https://simplabs.com and other sites on the Internet. <br> 
+          &rarr; Users may opt out of the use of the DART cookie by visiting the Google ad and content network privacy policy at the following URL - <a href="http://www.google.com/privacy_ads.html" title="Opt out of the Dart Cookie">http://www.google.com/privacy_ads.html</a> </p> 
+          <p><b>Our Advertising Partners</b><br>
```
Comment (marcoow) [inline imprint/index.html]: We don't have advertising partners.

File: imprint/index.html [hunk 2]
```diff
@@ -31,12 +31,49 @@
...
+          <ul></ul>
+          <p><em>While each of these advertising partners has their own Privacy Policy for their site, an updated and hyperlinked resource is maintained here: <a href="http://www.privacypolicyonline.com/privacy-policies">Privacy Policies</a>.<br /> 
+          You may consult this listing to find the privacy policy for each of the advertising partners of https://simplabs.com.</em></p>
+          <p> These third-party ad servers or ad networks use technology in their respective advertisements and links that appear on https://simplabs.com and which are sent directly to your browser. They automatically receive your IP address when this occurs. Other technologies (such as cookies, JavaScript, or Web Beacons) may also be used by our site's third-party ad networks to measure the effectiveness of their advertising campaigns and/or to personalize the advertising content that you see on the site. </p> 
+          <p> https://simplabs.com has no access to or control over these cookies that are used by third-party advertisers. </p> 
+          <p> </p><p><b>Third Party Privacy Policies</b><br>
```
Comment (marcoow) [inline imprint/index.html]: No Third Party ad servers involved on our site.

File: imprint/index.html [hunk 3]
```diff
@@ -31,12 +31,49 @@
...
+          You should consult the respective privacy policies of these third-party ad servers for more detailed information on their practices as well as for instructions about how to opt-out of certain practices. https://simplabs.com's privacy policy does not apply to, and we cannot control the activities of, such other advertisers or web sites. You may find a comprehensive listing of these privacy policies and their links here: <a href="http://www.privacypolicyonline.com/privacy-policy-links" title="Privacy Policy Links">Privacy Policy Links</a>.</p> 
+          <p> If you wish to disable cookies, you may do so through your individual browser options. More detailed information about cookie management with specific web browsers can be found at the browsers' respective websites. <a href="http://www.privacypolicyonline.com/what-are-cookies">What Are Cookies?</a></p>
+
+          <p><strong>Children's Information</strong><br />We believe it is important to provide added protection for children online. We encourage parents and guardians to spend time online with their children to observe, participate in and/or monitor and guide their online activity.
```
Comment (marcoow) [inline imprint/index.html]: I don't think this is relevant in our case.

File: imprint/index.html [hunk 4]
```diff
@@ -31,12 +31,49 @@
...
+
+          <p>
+          <b>Online Privacy Policy Only</b><br />
+          This privacy policy applies only to our online activities and is valid for visitors to our website and regarding information shared and/or collected there.
+          This policy does not apply to any information collected offline or via channels other than this website.</p>
+          <p><b>Consent</b><br />
+          By using our website, you hereby consent to our privacy policy and agree to its terms.
+          </p><br /><br /><p><b>Update</b><br />This Privacy Policy was last updated on: Monday, July 3rd, 2017.
+          <a href="http://www.PrivacyPolicyOnline.com" title="PrivacyPolicyOnline.com Approved Site" target="_blank"><img src="http://www.privacypolicyonline.com/images/privacypolicyonline-seal.png" border="0" alt="Privacy Policy Online Approved Site" align="right" /></a><br /><em>Should we update, amend or make any changes to our privacy policy, those changes will be posted here.</em>
```
Comment (marcoow) [inline imprint/index.html]: I'd prefer a solution where we don't have to show something like this.

**Comment by trabaldini** [comment]
fyi looked the laws up here:
https://www.bussgeldkatalog.org/datenschutzerklaerung-website/
https://www.gesetze-im-internet.de/tmg/__13.html
https://marketing.gelbeseiten.de/Online-Marketing/Website/Der-Cookie-Hinweis-auf-der-Website-Rechtliche-Pflicht-oder-nur-Lex-Google

**Comment by marcoow** [comment]
we'll go with a simpler solution

**Pull request closed without merging**